README.md

```diff
@@ -84,17 +92,18 @@ Whereas for CLN nodes, the following information is required:
 **Note that node addresses must be declare with HTTPS transport, i.e. <https://ip-or-domain:port>**
 
 Payment activity can be simulated in two different ways:
-* Random activity: generate random activity on the `nodes` provided, 
+
+- Random activity: generate random activity on the `nodes` provided,
   using the graph topology to determine payment frequency and size.
-* Defined activity: provide descriptions of specific payments that 
+- Defined activity: provide descriptions of specific payments that
   you would like the generator to execute.
 
 ### Setup - Random Activity
 
-To run the simulator with random activity generation, you just need to 
-provide a set of nodes and the simulator will generate activity based 
-on the topology of the underlying graph. Note that payments will only 
-be sent between the `nodes` that are provided so that liquidity does 
+To run the simulator with random activity generation, you just need to
```

Comment:
Please pull wholesale formatting changes out into their own commit

Reply:
This still hasn't been addressed?

If you want to run all these formatting changes on the README, please run them first and then make your changes.